### PR TITLE
ENH - re-implementation of (undocumented) patchsvd and lbex options

### PR DESCRIPTION
**Pull request by schoffelen**
patch svd now allows atlas in input

### CODE REVIEW

**Comment by schoffelen** [comment]
This might be relevant for @sjeung and @reneterporten

**Comment by schoffelen** [comment]
I need to still make a test function, so stay tuned

File: test/test_lbex.m [hunk 1]
```diff
@@ -0,0 +1,228 @@
+function test_patchsvd(datadirs)
```
Comment (robertoostenveld) [inline test/test_lbex.m:1]: the function name here and in the other test script are swapped

Comment (schoffelen) [reply test/test_lbex.m:1]: oeps

**Pull request merged**